pandas/core/indexes/multi.py

```diff
@@ -735,13 +735,14 @@ def array(self):
     def dtypes(self) -> Series:
         """
         Return the dtypes as a Series for the underlying MultiIndex.
+
+        .. versionchanged:: 1.4.0
```

Comment:
I don't think that we need this, since this qualifies as a bug fix

Reply:
agree

Reply:
Done